.github/workflows/standard_spa_pr_NPM.yml

```diff
@@ -4,11 +4,10 @@ on:
   workflow_call:
     inputs:
       working_directory:
-        description:
-          'The working directory from which to run the workflow commands'
+        description: "The working directory from which to run the workflow commands"
```

Comment:
I installed the official GH VS CODE actions plugin and it reformated the full page